coinstacks/bitcoin/ingester/src/parseTx/index.ts

```diff
@@ -0,0 +1,54 @@
+import { BigNumber } from 'bignumber.js'
+import { Tx, Vin, Vout } from '@shapeshiftoss/blockbook'
+import { ParseTx, TxTransfer, Token } from '@shapeshiftoss/common-ingester'
+
+// keep track of all individual tx components and add up the total value transferred
+const aggregateTransfer = (transfer: TxTransfer, value: string, token?: Token): TxTransfer => {
+  if (transfer) {
+    transfer.totalValue = new BigNumber(transfer.totalValue).plus(new BigNumber(value)).toString(10)
+    transfer.components.push({ value: value })
+  } else {
+    transfer = { totalValue: value, components: [{ value: value }] }
+  }
+
+  return { ...transfer, token }
+}
+
+export const parseTx = async (tx: Tx, address: string): Promise<ParseTx> => {
+  const pTx: ParseTx = {
+    ...tx,
+    address,
+    receive: {},
+    send: {},
+  }
+
```

Comment:
don't need bignumber, can use Number() and isNaN()

Reply:
example of send :
```
{"BTC":{"totalValue":"22618520","components":[{"value":"22618520"}]}}
```

Reply:
move from symbols to unique asset IDs

Reply:
- sticking with Big Number for consistency
- created separate issue for unique asset IDs